pkg/antctl/transform/member_token/transform.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package member_token
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	"antrea.io/antrea/pkg/antctl/transform/common"
+)
+
+type Response struct {
+	Name string `json:"name" yaml:"name"`
+}
+
+func Transform(r interface{}, single bool) (interface{}, error) {
+	if single {
+		return listTransform([]corev1.Secret{r.(corev1.Secret)})
+	}
+	return listTransform(r)
+}
+
+func listTransform(l interface{}) (interface{}, error) {
+	secrets := l.([]corev1.Secret)
+	var result []interface{}
+
+	for i := range secrets {
+		item := secrets[i]
+		response := Response{
+			Name: item.Name,
```

Comment:
Please add Namespace in the output, it will be useful when user is trying to list all tokens from all Namespaces.

Reply:
@jianjuns Hi Jianjun, I think we need a output of Namespace in case that two different token have the same name in different namespaces?